identity/app/services/ExactTargetService.scala

```diff
@@ -0,0 +1,44 @@
+package services
+
+import com.exacttarget.fuelsdk.{ETResponse, ETSubscriber}
+import utils.SafeLogging
+import common.ExecutionContexts
+import conf.IdentityConfiguration
+import scala.concurrent.Future
+import scalaz.\/
```

Comment:
Why not use cats here for consistency with other parts of the codebase. `Either(T)` seems like a better choice. As well as consistency this means you're using the stdlib's Either, which is much more accessible.